src/Files.App/UserControls/Menus/FileTagsContextMenu.cs

```diff
@@ -62,7 +62,7 @@ private void TagItem_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
 			}
 		}
 
-		private void Item_Opening(object sender, object e)
+		private void Item_Opening(object? sender, object e)
```

Comment:
This doesn't help to prevent exceptions. I think we need to consider the case where `SelectedItems` is null.